src/app/config/confparse.c

```diff
@@ -550,13 +550,52 @@ config_var_is_gettable(const config_var_t *var)
   return true;
 }
 
-bool
-config_var_is_contained(const config_var_t *var)
+/**
+ * Return true iff this is variable is "derived" from another -- that is,
```

Comment:
Typo, consistency:
```
this is variable
this variable <b>var</b>
```

Reply:
This documentation no longer exists by the last commit in the branch: "derived" is replaced by NOCOPY and NOCMP in 4d78692e09bc7503bdac6b0c0a0e1bf00f951eac